gtp2ogs.js

```diff
@@ -418,34 +418,84 @@ process.on('uncaughtException', function (er) {
   }
 })
 
+// console messages : for bot admins information
+
+console.log("You are using gtp2ogs version 6.0 \n - For changelog or devel updates, please visit : https://github.com/online-go/gtp2ogs/tree/devel \n - For known issues, please visit https://github.com/online-go/gtp2ogs/KNOWN-ISSUES.md");
```

Comment:
Remove the extra space after "visit" in `please visit :`

Reply:
done :1st_place_medal: